Description: Fix incorrect type in set_timerslack.
 set_timerslack passes a u64 to a variadic function where it should pass
 a c_ulong. This causes breakage on some 32-bit architectures, notablly arm32
 eabi.

 This patch adds a cast to fix it.
Author: Peter Michael Green <plugwash@debian.org>

--- rust-nix-0.27.1.orig/src/sys/prctl.rs
+++ rust-nix-0.27.1/src/sys/prctl.rs
@@ -160,7 +160,7 @@ pub fn get_name() -> Result<CString> {
 /// Sets the timer slack value for the calling thread. Timer slack is used by the kernel to group
 /// timer expirations and make them the supplied amount of nanoseconds late.
 pub fn set_timerslack(ns: u64) -> Result<()> {
-    let res = unsafe { libc::prctl(libc::PR_SET_TIMERSLACK, ns, 0, 0, 0) };
+    let res = unsafe { libc::prctl(libc::PR_SET_TIMERSLACK, ns as c_ulong, 0, 0, 0) };
 
     Errno::result(res).map(drop)
 }
